### DISCUSSION TITLE
Add 404 icon for GitHub Pages

### DISCUSSION THREAD

**Discussion by PossibleReference** [community/community#30985] [Other Feature Feedback, Questions, & Ideas] [answered]
Right now, the 404 page for GitHub Pages looks plain and I want it to represent GitHub as it is.

Maybe like the cat of GitHub in the server room and there is wires everywhere.

**Comment by thuongtruong1009** [comment, marked as answer]
I think that GitHub does not add the 404 icon to the not found page is because compared to normal websites, GitHub will also appear a not found page when that user's account is set to private or the user entered it incorrectly. data or block someone, someone else accessing this person's account will encounter a not found page. So besides the 404 function, it will probably also be a 401 or 400, 403 page, so Github won't put the 404 icon for all of that.
Hope it helps you
